src/components/GradeImage/GradeImage.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import Card from "./GradeImage";
+import { Grade } from "../../api/services/types";
+
+const meta: Meta<typeof Card> = {
+  component: Card,
+  args: {
+    grade: Grade.SEED,
+    width: 105,
+    height: 128,
+  },
+};
+
+export default meta;
+
+export const Seed: StoryObj<typeof Card> = {
+  args: {
+    grade: Grade.SEED,
+    width: 105,
+    height: 128,
+  },
```

Comment:
```suggestion
argTypes: {
    grade: {
      options: [
        Grade.SEED,
        Grade.SPROUT,
        Grade.LEAF,
        Grade.BRANCH,
        Grade.FRUIT,
        Grade.TREE,
      ],
      control: { type: "select" },
    },
  },
```
  
미팅에서 input으로 핸들링하는게 보기 좀 그렇다고 하셨는데, select로 핸들링하는 방법은 어떨까요?
[storybook controls]( https://storybook.js.org/docs/essentials/controls)

Reply:
이 방법은 타입스크립트를 안 쓰는 프로젝트에서는 유용한데요. 타입스크립트를 쓰는 프로젝트에서는 이상적인 접근이 아닌 것 같아요. 스토리북에 자동으로 grade의 argTypes를 추론할 수 있어야 하는데, 하필 grade가 enum 타입이라서 문자열로 잘못 추론되고 있어요. grade의 타입을 union으로 변경해주면 굳이 이렇게 스토리만을 위해서 argTypes을 명시하고 관리하지 않아도 해결이 될텐데 어떻게 생각하세용?

Reply:
오호 options의 타입이 `any[] | undefined` 로 추론되는군요..?
@DaleSeo님 코멘트보고 union타입으로 GradeImageProps의 grade타입을 유니온으로 변경해봤는데 동일하게 `any[] | undefined`로 추론되는걸 확인했어요. 
그런점에서 타입추론이 되지않는 argTypes를 사용해야하는 의문점과 argTypes는 자바스크립트에서 유용하다는건 새롭게 깨닫게 됐어요!
저는 현재 상황에서 typesafe하게 사용하려면 각 grade에 따른 스토리를 작성해야한다고 생각해요.

그런데 여러개의 컴포넌트를 합성하여 새로운 컴포넌트를 만들어 스토리를 만드는 경우는 다를거 같아요.
각 컴포넌트의 타입에 따른 경우의수와 다른 컴포넌트와 조합이 됐을 때 경우의수는 방대해지기에 타입추론이 되지 않는 argTypes를 사용해야하는 날은 올거 같다는 개인적인 의견입니다.😶

Reply:
아 유니온이라면 혹시 배열로 나열하는것을 의미하는 걸 까요?
그에 대안이라면 Object.keys를 사용하는건 어떨까요?
```ts
argTypes: {
    grade: {
      control: "select",
      options: Object.keys(Grade),
    },
  },
```

Reply:
> 그런데 여러개의 컴포넌트를 합성하여 새로운 컴포넌트를 만들어 스토리를 만드는 경우는 다를거 같아요.
각 컴포넌트의 타입에 따른 경우의수와 다른 컴포넌트와 조합이 됐을 때 경우의수는 방대해지기에 타입추론이 되지 않는 argTypes를 사용해야하는 날은 올거 같다는 개인적인 의견입니다.😶

네네, 충분히 일리가 있는 의견이십니다. 그런 날이 오면 다시 얘기해보시죠 😉

> 아 유니온이라면 혹시 배열로 나열하는것을 의미하는 걸 까요?
그에 대안이라면 Object.keys를 사용하는건 어떨까요?

에궁... 제 설명이 제대로 전달이 안 된 것 같습니다. 요점은 스토리북이 추론해주는 argTypes를 덮어쓰는 것은 가급적 피하는 것이 좋다는 것입니다. 덮어쓰는 순간 컴포넌트 함수의 입력 타입과 스토리의 argTypes가 일치된 상태로 유지해야하는 오버헤드가 발생하거든요.

원래 티켓의 목표에서 벗어나는 주제로 PR 쓰레드가 너무 길어지고 있는 것 같아서 본 PR은 병합을 하도록 할께요. 그 다음, 제가 이상적이라고 생각하는 방식으로 코드를 수정해서 별도의 PR을 한번 올리고 리뷰 요청을 드리겠습니다.

Reply:
이슈: #157